components/tekton/pipelineruns/run-sync-test-suite/run-test-suite.yaml

```diff
@@ -0,0 +1,23 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: run-vse-sync-testsuite-run
+  labels:
+    tekton.dev/pipeline: run-vse-sync-testsuite
+spec:
+  pipelineRef:
+    name: run-vse-sync-testsuite
+  serviceAccountName: pipeline-credentials-sa
+  timeout: 1h0m0s
+  workspaces:
+    - name: report
+      volumeClaimTemplate:
```

Comment:
I think that using `volumeClaimTemplate` will create a PVC for every run, maybe worth to use `volumeClaim` instead?

Reply:
Yes it will but I wanted to make sure there was a clean environment each time. If will the files from previous runs still exist? If so we `volumeClaim` should we make sure that we remove any output files.

Reply:
Good point, I'm just not sure what will happen when we'll have a lot of `PipelineRuns`. The PVC is deleted when the `PipelineRun` is deleted so we will have a lot of PVCs
Maybe we can add something like `deleteExisting` variable in the `git-clone` task?

Reply:
Good question. We will need some for GC on the runs anyway so deleting the runs will also clean up the volume claims. The volume claims just make it more important that we have GC on the runs.